Rakefile

```diff
@@ -23,13 +23,15 @@ end
 
 desc 'Setup and run all tests'
 task :setup_and_run_tests do
-  ENV['BUNDLE_GEMFILE'] = File.join(__dir__, 'gemfiles', "#{ENV['RAILS_VERSION']}.gemfile") if ENV['RAILS_VERSION']
+  rails_version = ENV['RAILS_VERSION']
 
-  unless ENV['BUNDLE_GEMFILE']
-    puts 'You need to specify either BUNDLE_GEMFILE pointing to a Gemfile in gemfiles/ directory or RAILS_VERSION or the tests will fail.'
-    exit 1
+  unless ENV['RAILS_VERSION']
+    rails_version = Dir.glob(File.join(__dir__, 'gemfiles', '*.gemfile')).max[%r{gemfiles/(.*?)\.gemfile}, 1]
+    puts "RAILS_VERSION has not been specified, defaulting to the latest available Rails version (#{rails_version})"
   end
 
+  ENV['BUNDLE_GEMFILE'] = File.join(__dir__, 'gemfiles', "#{rails_version}.gemfile")
```

Comment:
interesting

Reply:
I have switch multiple projects to remove all of the individual gemfiles and use ENV var inside the main gemfiles instead... for some reason, it fails for this project. but I have this WIP branch if you want to take a look 

https://github.com/mileszs/wicked_pdf/compare/master...mathieujobin:wicked_pdf:ruby33